sentry/src/event_aggregator.rs

```diff
@@ -0,0 +1,127 @@
+use crate::access::check_access;
+use crate::db::event_aggregate::insert_event_aggregate;
+use crate::db::DbPool;
+use crate::event_reducer;
+use crate::Application;
+use crate::ResponseError;
+use crate::Session;
+use async_std::sync::RwLock;
+use chrono::Utc;
+use primitives::adapter::Adapter;
+use primitives::sentry::{Event, EventAggregate};
+use primitives::{Channel, ChannelId};
+use slog::{error, Logger};
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::time::Duration;
+use tokio::time::delay_for;
+
+pub(crate) type Aggregate = Arc<RwLock<HashMap<ChannelId, EventAggregate>>>;
+
+#[derive(Default, Clone)]
+pub struct EventAggregator {
+    aggregate: Aggregate,
+}
+
+pub fn new_aggr(channel_id: &ChannelId) -> EventAggregate {
+    EventAggregate {
+        channel_id: channel_id.to_owned(),
+        created: Utc::now(),
+        events: HashMap::new(),
+    }
+}
+
+async fn store(db: &DbPool, channel_id: &ChannelId, logger: &Logger, aggr: Aggregate) {
+    let mut recorder = aggr.write().await;
+    let ev_aggr: Option<&EventAggregate> = recorder.get(channel_id);
+    if let Some(data) = ev_aggr {
+        if let Err(e) = insert_event_aggregate(&db, &channel_id, data).await {
+            error!(&logger, "{}", e; "eventaggregator" => "store");
```

Comment:
`"module" => "event_aggregator", "in" => "store"`

Reply:
what's the difference?

Reply:
consistency?

Reply:
I prefer the short form

Reply:
Then make an issue to fix _all_ the other places to make consistent + at least use `snake_case` so that we don't scratch our heads all the time

Reply:
https://github.com/AdExNetwork/adex-validator-stack-rust/blob/dev/sentry/src/main.rs#L148

Reply:
As I said - we should make them consistent. Also for the main, well it's not actually a module, it's the binary entry point. I am welcoming changes. Although I think a bit more verbosity is good for the logging if in the future we start logging other stuff (which are not modules, etc.) to keep us away from confusion